src/main/java/com/rocketden/main/service/ProblemService.java

```diff
@@ -176,7 +181,7 @@ public ProblemDto deleteProblem(String problemId) {
             problems = repository.findAllByDifficulty(difficulty);
         }
 
-        if (problems == null || problems.isEmpty()) {
+        if (problems == null) {
```

Comment:
Since users can now choose problems, there could be situations in which they choose a problem for the room and also have selected a difficulty that doesn't have any valid problems. Before, this would cause an exception to be thrown, but by removing it, it will let is slide as long as there exists 1 problem for the game.

Reply:
Ah I got it - if I understand correctly though, the only time that `problems == null` is if there is some sort of internal error, because as the code stands it should always at least just be an empty list. Should we change the error here to something akin to "Internal Server Error", or just leave it as "Not Found"?

Reply:
Sure, I can change it to be an internal error - done!